ckanext/cioos_theme/templates/header.html

```diff
@@ -87,8 +87,10 @@ <h2>{{ gns.site_description }}</h2>
     {% block header_site_navigation %}
     <nav>
       <ul>
-        {% block header_site_navigation_tabs %} {{ h.build_nav_main( ('search', _('Datasets')), ('organizations_index', _('Organizations')),
-        ('group_index', _('Groups')), ('home.about', _('About')) ) }} {% endblock %}
+        {% block header_site_navigation_tabs %}
+        <li><a href="{{ g.site_home_url }}">{{ _('HOME') }}</a></li>
+        {{ h.build_nav_main( ('search', _('Datasets')), ('organizations_index', _('Organizations')),
+        ('group_index', _('Groups')), ('home.about', _('About')), ('home.index', _('Data Catalogue')) ) }} {% endblock %}
```

Comment:
For home.index here - perhaps the hardcoded 'Data Catalogue' text could be replaced with a new element from the config to specify it?

Reply:
we could. I'm hesitant to make this a config option as I think it would be better to hardcode to promote uniformity in the interface between RA's. I'm not attached to any particular wording so am happy to use what ever makes sense for the whole. We can also just remove this link if it's not useful. We may need a review of the CKAN interface by people with more UI expertise then I. I'm happy with your suggestion of 'Catalogue Home' in #7 and can make that change. If you are ok with it, we can go with that for now and do a more formal UI review later.

Reply:
I agree a UI review for this change by e.g. the User Experience team would be great. But for now going with 'Catalogue Home' sounds good to me.